package.json

```diff
@@ -55,5 +55,30 @@
     "build": "npm install && grunt",
     "start": "grunt watch",
     "test": "grunt"
+  },
+  "commitplease": {
+    "components": [
+      "Docs",
+      "Tests",
+      "Build",
+      "Release",
```

Comment:
What's this one? I think for UI and QUnit we dump release-related commits into "Build".